jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -727,12 +727,14 @@ private InterpreterResult executeSql(String dbPrefix, String sql,
     try {
       List<String>  sqlArray = sqlSplitter.splitSql(sql);
       for (String sqlToExecute : sqlArray) {
-        LOGGER.info("Execute sql: " + sqlToExecute);
-        if (sqlToExecute.trim().toLowerCase().startsWith("set ")) {
+        if (sqlToExecute.trim().toLowerCase().startsWith("set ") ||
+                sqlToExecute.trim().toLowerCase().startsWith("list ") ||
+                sqlToExecute.trim().toLowerCase().startsWith("add ")) {
```

Comment:
It would be good to move `sqlToExecute.trim().toLowerCase()` out of `if` statement.
Anyway, which sql language has `list` keyword?

Reply:
Hive sql: `list jars`

Reply:
As [a Hive Manual](https://cwiki.apache.org/confluence/display/hive/languagemanual+cli), It seems be a commend of the interpreter shell mode. There is other command like `DELETE` also.

Reply:
That's right @cuspymd Let me complete the full list.